Tribler/Test/GUI/test_gui.py

```diff
@@ -494,6 +494,16 @@ def test_debug_pane(self):
         self.wait_for_list_populated(window.debug_window.threads_tree_widget)
         self.screenshot(window.debug_window, name="debug_panel_threads_tab")
 
+        # Logs shown in ui and from the debug endpoint should be same
+        window.debug_window.debug_tab_widget.setCurrentIndex(6)
+        QTest.qWait(2000)
```

Comment:
Please don't wait for two whole seconds. Instead, you should have an event in Qt (something like `log_ready` and listen for that. The reason I'm waiting in line 508 is that matplotlib needs some time for drawing the graph.